chapters/cli/exit.rst

```diff
@@ -1,8 +1,25 @@
+====================
```

Comment:
Could you also add a table of contents? :)
.. contents:: Table of Contents
    :depth: 2

Reply:
✔️